types/testing-library__cypress/tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "semicolon": [
```

Comment:
Please open new PR removing all the rules here and fix errors. We do not want new definition with custom rules. If the rule is absolutely necessary disable it at the location, instead of for whole package, so we can review correctness of it.